day_two_guide/topics/increasing_docker_storage.adoc

```diff
@@ -127,6 +141,74 @@ ose-app-node01.example.com   Ready                      24m       v1.6.1+5115d70
 ose-app-node02.example.com   Ready                      24m       v1.6.1+5115d708d7
 ----
 
+*Increasing storage with a new disk*
+
+. Evacuate the node xref:evacuating-a-node[following the steps above].
+
+. Stop the `docker` and `atomic-openshift-node` services:
++
+----
+# systemctl stop docker atomic-openshift-node
+----
+
+. Resize the existing disk as desired. This can can depend on your environment:
++
+* If you are using LVM (Logical Volume Manager)::
++
+** link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/logical_volume_manager_administration/lv#LV_remove[Remove the logical volume]:
++
+----
+# lvremove /dev/docker_vg/docker/lv
+----
++
+** link;https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/logical_volume_manager_administration/vg_admin#VG_remove[Remove the Docker volume group]:
++
+----
+# vgremove docker_vg
+----
++
+** link;https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/logical_volume_manager_administration/physvol_admin#PV_remove[Remove the physical volume]:
++
+----
+# pvremove /dev/<my_previous_disk_device>
+----
++
+* If you are using a cloud provider, you can detach the disk,
+destroy the disk, then create a new bigger disk and attach it to the instance.
++
+* For any other non-cloud environment, the disk and filesystem can be resized. See the
```

Comment:
s/For any other non-cloud environment, the disk and filesystem can be resized./For non-cloud environments, you can resize the disk and file system.
(Style says that "file system" is two words.)